### PR TITLE
change deployment list titles

### PR DESCRIPTION
**Pull request by 0oM4R**
### Description

Describe the changes introduced by this PR and what does it affect
the new titles 
`Clusters`,
`CapRover machines`,
`Discourse machines`,
`Funkwhale machines`,
`Mastodon machines`,
`Mattermost machines`,
`Owncloud machines`,
`Peertube machines`,
`Subsquid machines`,
`Taiga machines`,
`Wordpress machines`,
`Gateway names`,
`TFhubValidator machines`,
`Casperlabs machines`,
`Presearch machines`,
`QSFS virtual machines`,
`Micro Virtual machines`,
`Node Pilot machines`,
`Full virtual machines`,
`Algorand machines`,
`Umbrel machines`,

image

> Some of them are not used, but we should have them to avoid any error while accessing the `deplomentTitle` with the project name
### Changes

List of changes this PR includes

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/532

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by xmonader** [comment]
Virtual Machines
Kubernetes, and caprover use Cluster suffix
Everything else can be X instances

File: packages/playground/src/types/index.ts [hunk 1]
```diff
@@ -57,7 +57,27 @@ export enum ProjectName {
...
+  Discourse: "Discourse Instances",
+  Funkwhale: "Funkwhale Instances",
+  Mastodon: "Mastodon Instances",
+  Mattermost: "Mattermost Instances",
+  Owncloud: "Owncloud Instances",
+  Peertube: "Peertube Instances",
+  Subsquid: "Subsquid Instances",
+  Taiga: "Taiga Instances",
+  Wordpress: "Wordpress Instances",
+  TFhubValidator: "TFhubValidator Instances",
```
Comment (Mahmoud-Emad) [inline packages/playground/src/types/index.ts:72]: We removed the validator no?

Comment (0oM4R) [reply packages/playground/src/types/index.ts:72]: should we have only the current active weblets ? 
as qvm should be removed to

Comment (xmonader) [reply packages/playground/src/types/index.ts:72]: QVM should also be hidden too

File: packages/playground/src/weblets/tf_deployment_list.vue [hunk 2]
```diff
@@ -1,12 +1,7 @@
 <template>
   <weblet-layout ref="layout">
     <template #title>
-      Deployment List
-      {{
-        $props.projectName
-          ? "(" + (tabs.find(tab => tab.value === $props.projectName)?.title ?? $props.projectName) + ")"
-          : ""
-      }}
+      {{ title }}
```
Comment (MohamedElmdary) [inline packages/playground/src/weblets/tf_deployment_list.vue:4]:
```suggestion
      {{ title || "Deployment List" }}
```
Make title optional to allow to use component as before (might be needed later)

**Pull request merged**